code/game/objects/effects/spawners/random/food_or_drink.dm

```diff
@@ -0,0 +1,219 @@
+		/obj/item/reagent_containers/food/drinks/bottle/maltliquor = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/whiskey = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/gin = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/vodka = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/tequila = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/rum = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/vermouth = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/cognac = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/wine = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/kahlua = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/amaretto = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/hcider = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/absinthe = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/sake = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/grappa = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/applejack = 5,
+		/obj/item/reagent_containers/glass/bottle/ethanol = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/fernet = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/champagne = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/absinthe/premium = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/goldschlager = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/patron = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/kong = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/lizardwine = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/vodka/badminka = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/trappist = 1,
+	)
+
+/obj/effect/spawner/random/food_or_drink/pizzaparty
+	name = "pizza bomb spawner"
+	icon_state = "pizzabox"
+	loot = list(
+		/obj/item/pizzabox/margherita = 3,
+		/obj/item/pizzabox/meat = 3,
+		/obj/item/pizzabox/mushroom = 3,
+		/obj/item/pizzabox/bomb/armed = 1,
+	)
+
+/obj/effect/spawner/random/food_or_drink/seed_vault
+	name = "seed vault seeds"
+	icon_state = "seed"
+	loot = list(
+		/obj/item/seeds/gatfruit = 10,
+		/obj/item/seeds/cherry/bomb = 10,
+		/obj/item/seeds/berry/glow = 10,
+		/obj/item/seeds/sunflower/moonflower = 8,
+	)
+
+/obj/effect/spawner/random/food_or_drink/snack
+	name = "snack spawner"
+	icon_state = "chips"
+	loot = list(
+		/obj/item/food/spacetwinkie = 5,
+		/obj/item/food/cheesiehonkers = 5,
+		/obj/item/food/candy = 5,
+	  /obj/item/food/chips = 5,
+	  /obj/item/food/sosjerky = 5,
+		/obj/item/food/no_raisin = 5,
+		/obj/item/food/peanuts = 5,
+		/obj/item/food/cnds = 5,
+		/obj/item/food/energybar = 5,
+		/obj/item/reagent_containers/food/drinks/dry_ramen = 5,
+		/obj/item/food/peanuts/random = 3,
+		/obj/item/food/cnds/random = 3,
+		/obj/item/storage/box/gum = 3,
+		/obj/item/food/syndicake = 1,
+		/obj/item/food/peanuts/ban_appeal = 1,
+		/obj/item/food/candy/bronx = 1
+	)
+
+/obj/effect/spawner/random/food_or_drink/condiment
```

Comment:
I feel like vinegar and quality oil could fit here too, though I may be horribly biased

Reply:
Added with https://github.com/tgstation/tgstation/pull/60522/commits/1570e75ae4f2c188539d2f732ace9aa9153a3cc6